src/services/item/plugins/itemTag/schemas.ts

```diff
@@ -97,7 +95,10 @@ const getMany = {
   querystring: {
     allOf: [
       { $ref: 'http://graasp.org/#/definitions/idsQuery' },
-      { type: 'object', properties: { id: { type: 'array', maxItems: MAX_ITEMS_FOR_GET } } },
+      {
+        type: 'object',
+        properties: { id: { type: 'array', maxItems: MAX_TARGETS_FOR_READ_REQUEST } },
```

Comment:
You have swapped the constant, but the previous was not removed? What is the difference between the two?

Reply:
I did remove it? The difference is the number (15 instead of 30 now), but the important point is that it's synced with the constant used for GET /items as well. I needed a constant to refer in `query-client` to correctly split it.

Reply:
Thank you, my bad I missed the line